iOS/DoraemonKit/Src/Core/Util/Aspects/Aspects.h

```diff
@@ -0,0 +1,83 @@
+//
+//  Aspects.h
+//  Aspects - A delightful, simple library for aspect oriented programming.
+//
+//  Copyright (c) 2014 Peter Steinberger. Licensed under the MIT license.
+//
+
+#import <Foundation/Foundation.h>
+
+typedef NS_OPTIONS(NSUInteger, AspectOptions) {
+    AspectPositionAfter   = 0,            /// Called after the original implementation (default)
```

Comment:
`AspectOption*` instead of `AspectPosition*`?

Reply:
A little confused, this is a third party library code, we'd better keep the original code